nikola/post.py

```diff
@@ -251,20 +252,50 @@ def __init__(
                 list(set([x.strip() for x in _tag_list])),
                 alg=natsort.ns.F | natsort.ns.IC)
             self._tags[lang] = [t for t in self._tags[lang] if t]
-            if 'draft' in [_.lower() for _ in self._tags[lang]]:
-                is_draft = True
-                LOGGER.debug('The post "{0}" is a draft.'.format(self.source_path))
-                self._tags[lang].remove('draft')
 
-            if 'private' in self._tags[lang]:
-                is_private = True
-                LOGGER.debug('The post "{0}" is private.'.format(self.source_path))
-                self._tags[lang].remove('private')
+            status = self.meta[lang].get('status')
+            if status:
+                if status == 'published':
```

Comment:
I rewrote this, but not completely: I don't want to update `post_status` in case `status` has an invalid value. (Of course, we could turn this into an error. But it might screw up existing blogs which for whatever reason used `status` metadata already...)

Reply:
That would be me and the `projectpages` plugin (I guess I’ll make it `devstatus` from now on.)